src/main/java/frc/LoggyThings/LoggyThingManager.java

```diff
@@ -3,74 +3,98 @@
 
+/**
+ * The class that manages LoggyThings
+ */
 public class LoggyThingManager {
+    
     private static LoggyThingManager instance = null;
     private ArrayList<ILoggyMotor> mMotorList = new ArrayList<ILoggyMotor>();
     private EnumSet<ILoggyMotor.LogItem> mGlobalMaxLogLevel = ILoggyMotor.LogItem.LOGLEVEL_EVERYTHING;
     private long mMinGlobalLogPeriod = 0;// microseconds between possible logs, either user period or slowed down if
                                          // disabled
     private long mUserMinGlobalLogPeriod = 0; // user set min period, not overwritten if disabled
 
+    /**
+     * Starts a log file
+     */
     LoggyThingManager() {
-        System.out.println("Loggy Thing Manager V1.1 Initialized");
+        System.out.println("Loggy Thing Manager Initialized");
         DataLogManager.start();
         DriverStation.startDataLog(DataLogManager.getLog(), true);
     }
-    
+
+    /**
+     * 
+     * @return The microseconds between possible logs, either user period or slowed
+     *         down if disabled
+     */
     public long getMinGlobalLogPeriod() {
         return mMinGlobalLogPeriod;
     }
 
+    /**
+     * @param minLogPeriodSeconds seconds between possible logs
+     */
     public void setMinGlobalLogPeriod(double minLogPeriodSeconds) {
         mUserMinGlobalLogPeriod = ((long) (minLogPeriodSeconds * 1e6));
     }
 
+    /**
+     * 
+     * @param globalMaxLogLevel see {@link ILoggyMotor.LogItem}
+     */
     public void setGlobalMaxLogLevel(EnumSet<ILoggyMotor.LogItem> globalMaxLogLevel) {
         mGlobalMaxLogLevel = globalMaxLogLevel;
     }
 
+    /**
+     * 
+     * @return the max log level, see {@link ILoggyMotor.LogItem}
+     */
     public EnumSet<ILoggyMotor.LogItem> getGlobalMaxLogLevel() {
         return mGlobalMaxLogLevel;
     }
 
+    /**
+     * 
+     * @return Single running instance of {@link LoggyThingManager}
+     */
     public static LoggyThingManager getInstance() {
         if (instance == null)
             instance = new LoggyThingManager();
         return instance;
     }
 
-    public void registerLoggyMotor(ILoggyMotor loggyMotor) {
-        mMotorList.add(loggyMotor);
+    /**
+     * Add motor to list of devices to log
+     * 
+     * @param motor A {@link ILoggyMotor}
+     */
+    public void registerLoggyMotor(ILoggyMotor motor) {
+        mMotorList.add(motor);
     }
 
     boolean justFailed = false;
-    long lastDisabledTime=0;
-    // Call this from robot periodic, don't crash the robot
+
+    /**
+     * Call this from robot periodic, don't crash the robot
+     */
     public void periodic() {
         try {
             // Slow down logging if disabled
-            if(DriverStation.isEnabled()){
-                lastDisabledTime = WPIUtilJNI.getSystemTime();
-            }
-            mMinGlobalLogPeriod = ((WPIUtilJNI.getSystemTime()-10000000)<lastDisabledTime) ? mUserMinGlobalLogPeriod : 10000000;
```

Comment:
This is the special kind of disaster that probably means it shouldn't be removed unless you really know it isn't necessary. If I'd have to guess it is useful to keep the logging running at high rate for 10 seconds after disable so that we can see things coasting at the end of auton/at the end of the match. Because the robot goes disabled between auton and teleop